test/system/workflow_executions_test.rb

```diff
@@ -342,6 +344,34 @@ class WorkflowExecutionsTest < ApplicationSystemTestCase
     assert_selector 'div.random_seed-param > input[value="1"]'
   end
 
+  test 'can view workflow execution with samplesheet' do
+    workflow_execution = workflow_executions(:irida_next_example_completed)
+
+    visit workflow_execution_path(workflow_execution)
+
+    click_on I18n.t('workflow_executions.show.tabs.samplesheet')
+
+    assert_selector 'table tbody tr', count: 1
+    assert_text 'INXT_SAM_AAAAAAAAAA'
+    assert_text 'INXT_ATT_AAAAAAAAAA'
+    assert_text 'test_file.fastq'
+  end
+
+  test 'can view workflow execution with samplesheet with multiple files' do
+    workflow_execution = workflow_executions(:irida_next_example_completed_2_files)
+
+    visit workflow_execution_path(workflow_execution)
+
+    click_on I18n.t('workflow_executions.show.tabs.samplesheet')
+
+    assert_selector 'table tbody tr', count: 1
+    assert_text 'INXT_SAM_AAAAAAAAAA'
+    assert_text 'INXT_ATT_AAAAAAAAAA'
+    assert_text 'test_file.fastq'
+    assert_text 'INXT_ATT_AAAAAAAAAB'
+    assert_text 'test_file_A.fastq'
+  end
+
```

Comment:
Please add a test for a workflow execution with 0 samples.

Reply:
Now that you cannot launch a workflow without any samples this is not required.